### PR TITLE
Module 1

### PR DESCRIPTION
**Pull request by lindseyraven6**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -6,7 +6,32 @@
```
Comment (Jonesey13) [inline app.py:6]: You don't really need that submodule `DevOps-Course-Starter` listed above

File: session_items.py [hunk 2]
```diff
@@ -1,10 +1,6 @@
 from flask import session
```
Comment (Jonesey13) [inline session_items.py:1]: Empty file above can be removed

File: session_items.py [hunk 3]
```diff
@@ -67,3 +63,53 @@ def save_item(item):
...
+
+
+    return todo_id
+
+
+def complete_todo(id):
+  
+    existing_items = get_items()
+
+    for item in range(len(existing_items)):
```
Comment (Jonesey13) [inline session_items.py]: This code could be simplified by removing the range:
```python
for item in existing_items:
        if item['id'] == int(id):
            item['status'] = "Completed"
            break
```

File: templates/index.html [hunk 4]
```diff
@@ -1,18 +1,65 @@
...
+        <!-- ADD fORM -->
+
+        <div class="col-md-12 text-center">
+          <form action="/add-todo" method="POST">
+            <br/>
+
+            <span style='font-size:25px;'>&#9997;</span>
+              <label style="text-decoration-color: #fcdeee" for="name">Enter your to-do name:</label>
+              <input class="text-info" type="text" name="name" id="name">
+            <input class="text-info" type="submit" class="btn btn-dark btn-md" style="color:#b41369 !important" value="Submit"
```
Comment (Jonesey13) [inline templates/index.html]: This `<input>` element is missing a `>` (this currently breaks the submit button)

**Review by Jonesey13 — APPROVED** [review]
Looks good! This just manages to pass the project exercise (even though the submit button at the bottom doesn't work you can create todos by pressing enter)

I've left a few comments below but feel free to merge this PR in when you are ready 👍

**Comment by lindseyraven6** [comment]
This submit button works for me??? Strange!! Let me review it again! I’ve
never had a problem with the submit. I’ll take a look and get back to you.

Kind regards,

Lindsey

File: templates/index.html (hunk 4 above)
Comment (lindseyraven6) [reply templates/index.html]: > This `<input>` element is missing a `>` (this currently breaks the submit button)

cheers! must had deleted it by mistake and not realises. > has now been added back in. I need to learn to expand my VS code that way I can see everything all at once.

File: session_items.py (hunk 2 above)
Comment (lindseyraven6) [reply session_items.py:1]: if I remove the 'from flask import session' it returns with errors??

File: app.py (hunk 1 above)
Comment (lindseyraven6) [reply app.py:6]: how do I remove it?

**Pull request merged**